woomasonry.php

```diff
@@ -1,23 +1,17 @@
 <?php
 
-/**
- *
- * @link              http://www.intensewp.com/
- * @since             1.0.0
- * @package           Woomasonry
- *
- * @wordpress-plugin
- * Plugin Name:       WooCommerce Products Masonry Grid
- * Plugin URI:        http://www.intensewp.com/woomasonry-woocommerce-products-post-masonry-grid/
- * Description:       Display your products in masonry grid view. beautiful animation effect when changing the product categories
- * Version:           1.2
- * Author:            Intense WP
- * Author URI:        http://www.intensewp.com/
- * License:           GPL-2.0+
- * License URI:       http://www.gnu.org/licenses/gpl-2.0.txt
- * Text Domain:       woomasonry
- * Domain Path:       /languages
- */
+/*
+  Plugin Name:       WooCommerce Products Masonry Grid
+  Plugin URI:        https://draftpress.com/woocommerce-products-masonry-grid
```

Comment:
wrong url..

Reply:
we are not moving the wpintense plugins to draftpress so lets keep the URLs as is